config.yaml

```diff
@@ -2,10 +2,13 @@
 teracy-dev-k8s:
   # configurable here
   kubespray:
-    lookup_path: extensions # relative to the Vagrantfile
```

Comment:
I prefer keeping this as is:
- the input matches the location input of Gitsynch::sync
- do not break compatibility
- no reason to change